log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java

```diff
@@ -1,6 +1,6 @@
 package kcl.teamIndexZero.traffic.log;
 
-import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
+import kcl.teamIndexZero.traffic.log.FileIO.FileInput;
```

Comment:
Reverted back to originals, no errors now, thanks @An7ar35 and @lexaux                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the EE network.                                                                                                                                                                                                                From: Alexander CherednichenkoSent: Monday 21 March 2016 09:36To: teamIndexZero/index_zero_trafficsystemReply To: teamIndexZero/index_zero_trafficsystemCc: kumar awijeetSubject: Re: [index_zero_trafficsystem] Gui graphics kumar awijeet (#9)In log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java:
> @@ -1,6 +1,6 @@
>  package kcl.teamIndexZero.traffic.log;
>  
> -import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
> +import kcl.teamIndexZero.traffic.log.FileIO.FileInput;

okay okay wait a sec gentlemen - I remember that problem. 
@kumarawijeet pls do nothing on this one - I will sort it out myself; it is a weird bug in IDEA or something like that - it does not recognize things are changed from capital to lowercase and that's it. I'll fix it in Kumar's branch then.

—You are receiving this because you were mentioned.Reply to this email directly or view it on GitHub